src/pytorch_lightning/CHANGELOG.md

```diff
@@ -136,6 +136,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Fixed an issue to keep downscaling the batch size in case there hasn't been even a single successful optimal batch size with `mode="power"` ([#14372](https://github.com/Lightning-AI/lightning/pull/14372))
 
 
+- Squeezed tensor values while logging ([#14489](https://github.com/Lightning-AI/lightning/pull/14489))
```

Comment:
Is this correct:

```suggestion
- Squeezed tensor values when logging with `LightningModule.log` ([#14489](https://github.com/Lightning-AI/lightning/pull/14489))
```
I'd like to know where we are doing the squeezing, so mentioning self.log should be good?